requirements.txt

```diff
@@ -1,7 +1,9 @@
 pandas
-matplotlib
+# Workaround for https://github.com/mwaskom/seaborn/issues/3072
+matplotlib<3.6.1
 seaborn
 pyarrow
 scikit-learn
 folium
 ramp-workflow
+jupyter-lab
```

Comment:
It shouldn't be part of the requirements, because this is also used to create the env on the server. They should install jupyterlab separately (maybe it could be indicated in the readme).